pkg/client/k8s/k8s.go

```diff
@@ -285,10 +285,11 @@ func (k *K8sClient) GetStorageV1SCAsRaw(name string) (result []byte, err error)
 }
 
 // GetBatchV1JobAsRaw returns a Job instance
-func (k *K8sClient) GetBatchV1JobAsRaw(name string) (result []byte, err error) {
+func (k *K8sClient) GetBatchV1JobAsRaw(name, namespace string) (result []byte, err error) {
 	return k.cs.BatchV1().RESTClient().
 		Get().
 		Resource("jobs").
+		Namespace(namespace).
```

Comment:
I think k already has namespace field. Can we use that instead?

Reply:
Yes we can use that. Updated.